protoc-gen-gogo/generator/generator.go

```diff
@@ -1747,6 +1758,74 @@ func (g *Generator) GoType(message *Descriptor, field *descriptor.FieldDescripto
 	return
 }
 
+// GoMapDescriptor is a full description of the map output struct.
```

Comment:
Yeah I guess I'll just maintain this, since I also can't think of any easier way to do this.
Looks good :)